pkg/azurelustreplugin/entrypoint.sh

```diff
@@ -108,15 +111,36 @@ if [[ "${installClientPackages}" == "yes" ]]; then
     echo "deb [arch=amd64] https://packages.microsoft.com/repos/amlfs-${osReleaseCodeName}/ ${osReleaseCodeName} main" | tee /etc/apt/sources.list.d/amlfs.list
     apt-get update
   fi
-  
+
   echo "$(date -u) Installing Lustre client modules: ${pkgName}=${kernelVersion}"
 
-  # grub issue
-  # https://stackoverflow.com/questions/40748363/virtual-machine-apt-get-grub-issue/40751712
-  DEBIAN_FRONTEND=noninteractive apt install -y --no-install-recommends -o DPkg::options::="--force-confdef" -o DPkg::options::="--force-confold" \
-    ${pkgName}=${kernelVersion}
+  tries=3
+  install_success=false
+  while [[ tries -gt 0 ]]; do
+    # grub issue
+    # https://stackoverflow.com/questions/40748363/virtual-machine-apt-get-grub-issue/40751712
+    if ! DEBIAN_FRONTEND=noninteractive apt install -y --no-install-recommends -o DPkg::options::="--force-confdef" -o DPkg::options::="--force-confold" \
```

Comment:
I think you want apt-get, right?  To avoid the warning about an unstable apt API.

Reply:
This line doesn't change at all in this PR (just moved into the retry block. But I'll try changing it over because that message is annoying